microsoft/PowerToys
# [KBM Editor] Support WM_DPICHANGED event

## Summary of the Pull Request

**What is this about:**
The KBM Editor didn't support the WM_DPICHANGED event.

**What is include in the PR:** 
Add helper methods to the DPI aware lib. Cleanup existing methods.
Add support to WM_GETDPISCALEDSIZE and WM_DPICHANGED.

**How does someone test / validate:** 
On a single monitor: open the KBM editor window (test both key and shortcut editors) and while the window is open, go to the Windows Settings and change the current DPI scaling. The window should resize accordingly.
On a dual monitor setup: set different DPI scaling factors for the monitors, open the editor window on one monitor and move it to the other monitor and vice versa.`

## Quality Checklist

- [x] **Linked issue:** https://github.com/microsoft/PowerToys/issues/11014

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
